content/doc/book/installing/index.adoc

```diff
@@ -238,6 +238,50 @@ If your `/etc/init.d/jenkins` file fails to start Jenkins, edit the `/etc/defaul
 Here, "8081" was chosen but you can put another port available.
 ====
 
+===== Installation of Java
+
+Jenkins requires Java in order to run, yet certain distributions don't include this by default and  link:https://www.jenkins.io/doc/administration/requirements/java/[some Java version are incompatible] with jenkins.
+
+To install the Open Java Development Kit (OpenJDK) run the following:
```

Comment:
```suggestion
There are multiple Java implementations which you can use.
link:https://openjdk.java.net/[OpenJDK] is the most popular one at the moment, we will use it in this guide.
To install OpenJDK run the following:
```

Reply:
Hey @oleg-nenashev, thanks for your time. 
I'd added the note because it was required after the first review. It isn't a personal opnion. I tried to use the explanation from the official site  [manpages.ubuntu](http://manpages.ubuntu.com/manpages/trusty/man8/apt.8.html#differences%20to%20apt-get(8)) please check the link.

So what should I do with te note section? Keep it? change it? Remove it? 

I'm waitning for your answer.

Reply:
Did it. Thank you.